typescript/nomad-deploy/scripts/twoLocals/connect.ts

```diff
@@ -0,0 +1,82 @@
+import * as tom from '../../config/local/tom';
+import * as daffy from '../../config/local/jerry';
+import { ExistingCoreDeploy } from '../../src/core/CoreDeploy';
+import { deployEnvironment } from '../../src/chain';
+import { ExistingBridgeDeploy } from '../../src/bridge/BridgeDeploy';
+import { enrollSpoke } from '../../src/incremental';
+import { NomadContext } from '@nomad-xyz/sdk';
+
+let environment = deployEnvironment();
+
+const path =
+  process.env.DEPLOY_PATH || environment === 'staging'
+    ? '../../rust/config/staging'
+    : '../../rust/config/development';
+
+// Instantiate Governor Deploy Tom
+const tomConfig = environment === 'staging' ? tom.stagingConfig : tom.devConfig;
+const tomCoreDeploy = ExistingCoreDeploy.withPath(tom.chain, tomConfig, path);
+
+// Instantiate New Deploy, which is already existing at this moment
+let daffyConfig =
+  environment === 'staging' ? daffy.stagingConfig : daffy.devConfig;
+const daffyCoreDeploy = ExistingCoreDeploy.withPath(
+  daffy.chain,
+  daffyConfig,
+  path,
+);
+
+// Instantiate Existing Bridge Deploys
+const tomBridgeDeploy = new ExistingBridgeDeploy(
+  tom.chain,
+  tom.bridgeConfig,
+  path,
+);
+
+// Instantiate New Bridge Deploy, which is already existing at this moment
+const daffyBridgeDeploy = new ExistingBridgeDeploy(
+  daffy.chain,
+  daffy.bridgeConfig,
+  path,
+);
+
+const tomDomain = {
```

Comment:
make a `toSDK` helper function to generate this

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/ff75273178b6c024dbce8b40399362d8e5c5d680

Reply:
still need to move helper to suitable file